schema.graphql

```diff
@@ -40,10 +52,33 @@ type EpochCounter @entity {
   count: Int!
 }
 
-type Delegation @entity {
+interface Delegation {
+  # The delegatee address
+  id: ID!
+  totalWeight: BigInt!
+}
+
+type StakeDelegation implements Delegation @entity {
+  # The delegatee address
+  id: ID!
+  totalWeight: BigInt!
+  stakeDelegators: [StakeData!] @derivedFrom(field: "delegatee")
+}
+
+type TokenholderDelegation implements Delegation @entity {
+  # The delegatee address
   id: ID!
-  delegator: Bytes! # address
-  delegate: Bytes! # address
+  # The voting weight from the snapshot mechanism in the token and staking
+  # contracts. For Tokenholder DAO, there are currently two voting power
+  # sources:
+  # - Liquid T, tracked by the T token contract
+  # - Stakes in the T network, tracked  by the T staking contract. Note that
+  #   this also tracks legacy stakes (NU/KEEP); legacy stakes count for
+  #   tokenholders' voting power, but not for the total voting power of the
+  #   Tokenholder DAO."
```

Comment:
> legacy stakes count for tokenholders' voting power, but not for the total voting power of the Tokenholder DAO

Can you elaborate on this? I don't think it's correct but I'd like to see what's the reasoning behind it.

Reply:
I took this comment from [here](https://github.com/threshold-network/solidity-contracts/blob/main/contracts/governance/TokenholderGovernorVotes.sol#L33-L44) so I assume it's correct 🤔.

Reply:
Oh boy, that's funny. I wrote that 🤦  I just self-owned myself. 
OK, I understand now why I wrote it and it's indeed correct. Let's add this clarifying note here, and I'll add it to the contracts repo. 
```suggestion
  #   Tokenholder DAO (as it's already accounted by the Vending Machines)."
```

Reply:
1c6c5f6